### ISSUE TITLE
Show changelog/importat changes on website

### ISSUE THREAD

**Issue by RoyTakanen** [Testaustime/testaustime-frontend#59]
It would be amazing if the website showed changelog or important changes on some dedicated page. There is a public Github API endpoint where you can query last commits: https://api.github.com/repos/Testaustime/testaustime-backend/commits. If the "Merge pull request" are filtered it would be a good way to show the changelog. 

Also some sort of `changelog.md` could do (see: https://keepachangelog.com/en/1.0.0/).

**Comment by Eldemarkki** [comment]
Yes, I agree, this would be a good addition. I am strongly in favor of a CHANGELOG.md instead of a git log, because it will be less noisy and easier to read. I think I will start the changelog once we do a "1.0 release", because so far there hasn't been any versioning done, and writing a changelog without version numbers is a bit tricky.
